docs/upgrade.rst

```diff
@@ -0,0 +1,78 @@
+Upgrading
+#########
+
+We keep `a ChangeLog
+<https://github.com/spiral-project/ihatemoney/blob/master/CHANGELOG.rst>`_. Read
+it before upgrading.
+
+Ihatemoney follows `semantic versioning <http://semver.org/>`_. So minor/patch
+upgrades can be done blindly.
+
+General procedure
+=================
+
+*(sufficient for minor/patch upgrades)*
+
+1. From the virtualenv (if any)::
+
+    pip install -U ihatemoney
+
+2. Restart *supervisor*, or *Apache*, depending on your setup.
+
+You may also want to set new configuration variables (if any). They are
+mentioned in the `ChangeLog
+<https://github.com/spiral-project/ihatemoney/blob/master/CHANGELOG.rst>`_, but
+this is **not required for minor/patch upgrades**, a safe default will be used
+automatically.
+
+Version-specific instructions
+=============================
+
+*(must read for major upgrades)*
+
+When upgrading from a major version to another, you **must** follow special
+instructions:
+
+1.x → 2.x
+---------
+
+Switch from git installation to pip installation
+++++++++++++++++++++++++++++++++++++++++++++++++
+
+The recomended installation method is now using pip (git is now intended for
+development only).
+
+.. warning:: Be sure not to delete by accident your sqlite db or your settings
+             file if you stored it within the cloned folder.
+
+1. Delete the cloned folder
+
+
+.. note:: the following are to be run from your virtualenv, if any, (see
```

Comment:
nit: I would rephrase: "If you are using a virtualenv, then the following commands should be run inside it".